lib/AST/Module.cpp

```diff
@@ -1422,7 +1422,14 @@ bool SourceFile::hasPrivateImport(AccessLevel accessLevel,
                                 importPair.second.first.contains(
                                     ImportFlags::PrivateImport);
                        });
+  case AccessLevel::Open:
```

Comment:
This probably wasn't supposed to fall through here.

Reply:
there is a return before ``case AccessLevel::Open``

Reply:
Oops, sorry. Misleading partial diff!